src/main/resources/thrift/mypojo.thrift

```diff
@@ -0,0 +1,19 @@
+namespace java org.apache.flink.benchmark.thrift
```

Comment:
can we add a comment on top of the thrift file explaining how to generate definitions (copy paste commit message)?